dart/lib/src/sentry_options.dart

```diff
@@ -136,6 +136,15 @@ class SentryOptions {
   /// example : ['sentry'] will include exception from 'package:sentry/sentry.dart'
   List<String> get inAppIncludes => List.unmodifiable(_inAppIncludes);
 
+  /// Configures wether stack trace frames are considered in app frames as default.
+  /// You can use this to essentially make [inAppIncludes] or [inAppExcludes]
+  /// an allow or deny list.
+  /// This value is only used if Sentry can not find the origin of the frame.
+  ///
+  /// E.g. if [isStackFrameInAppDefault] is true you only need to maintain
+  /// [inAppExcludes].
+  bool isStackFrameInAppDefault = true;
```

Comment:
I feel like there could be a better name for this, though I can't think of any

Reply:
we don't add `is` to the other bools in this class, whats about:
`inAppFrames`only? or `inAppFramesInclusive` or `inAppInclusiveFrames`.
@bruno-garcia ?

Reply:
I'll throw more ideas but ultimately it's your call folks:
* `defaultInAppInclude` - As in, by default it's in app included.
* `preferInApp`

sorry, I don't feel like my suggestions are necessarily less crappy :)

Reply:
`considerStackFramesInApp` 😁

Note to self: Edit changelog accordingly